src/app/page.tsx

```diff
@@ -3,29 +3,25 @@ import { Button } from '@/common/components/Button';
 import { Heading } from '@/common/components/Heading';
 import { Main } from '@/common/components/Layout';
 
-import { TotoCard, } from '@/modules/toto/components/TotoCard';
+import { TotoCard } from '@/modules/toto/components/TotoCard';
 import { useTotoCards } from '@/modules/toto/hooks/useTotoCards';
 import { useTotoSubmissionModal } from '@/modules/toto/hooks/useTotoSubmissionModal';
+import { useUser } from '@/modules/toto/hooks/useUser';
 import { useUserCards } from '@/modules/toto/hooks/useUserCards';
 
 export default function Home() {
+    const userId = useUser();
     const { TotoCards } = useTotoCards();
     let USER_HAS_SUBMITTED_TICKET = false;
-    
+
     // user submitted ticket
-    const MY_SUBMITTED_TICKET = useUserCards();
-    if (MY_SUBMITTED_TICKET) {
+    const MY_SUBMITTED_TICKET = useUserCards(userId);
+    console.log(MY_SUBMITTED_TICKET);
+    if (MY_SUBMITTED_TICKET.userTickets.length > 0) {
```

Comment:
since USERHASSUBMITTEDTICKET is only used once, we might as well just use line20 as the logic. can add a comment to indicate the user has submitted tickets if you want.